xarray/tests/test_units.py

```diff
@@ -8,9 +8,16 @@
 import xarray as xr
 from xarray.core import dtypes, duck_array_ops
 
-from . import assert_allclose, assert_duckarray_allclose, assert_equal, assert_identical
+from . import (
+    assert_allclose,
+    assert_duckarray_allclose,
+    assert_equal,
+    assert_identical,
+    requires_dask,
+)
 from .test_variable import _PAD_XR_NP_ARGS
 
+dask = pytest.importorskip("dask")
```

Comment:
this should not be necessary with `requires_dask`, otherwise all `pint` tests will be skipped in the `all-but-dask` CI. Instead, I would import `dask.array` in the test.